examples/notebooks/use-cases/BBP KG Forge retrieval.ipynb

```diff
@@ -0,0 +1,167 @@
+      "______________________\n",
+      "Execution 2 id other bucket Cross bucket True Retrieve source True\n",
+      "rev 3\n",
+      "______________________\n",
+      "Execution 3 self other bucket Cross bucket True Retrieve source True\n",
+      "rev 3\n",
+      "______________________\n",
+      "Execution 4 id same bucket Cross bucket False Retrieve source True\n",
+      "rev 3\n",
+      "______________________\n",
+      "Execution 5 self same bucket Cross bucket False Retrieve source True\n",
+      "rev 3\n",
+      "______________________\n",
+      "Execution 6 id other bucket Cross bucket False Retrieve source True\n",
+      "<action> catch_http_error\n",
+      "<error> RetrievalError: resource 'http://purl.obolibrary.org/obo/GO_0038048' not found in project 'dke/kgforge'\n",
+      "\n",
+      "Not found\n",
+      "______________________\n",
+      "Execution 7 self other bucket Cross bucket False Retrieve source True\n",
+      "<action> retrieve\n",
+      "<error> RetrievalError: Provided resource identifier https://bbp.epfl.ch/nexus/v1/resources/neurosciencegraph/datamodels/_/http:%2F%2Fpurl.obolibrary.org%2Fobo%2FGO_0038048 is not inside the current bucket, use cross_bucket=True to be able to retrieve it\n",
+      "\n",
+      "Not found\n",
+      "______________________\n",
+      "Execution 8 id same bucket Cross bucket True Retrieve source False\n",
+      "rev 3\n",
+      "______________________\n",
+      "Execution 9 self same bucket Cross bucket True Retrieve source False\n",
+      "rev 3\n",
+      "______________________\n",
+      "Execution 10 id other bucket Cross bucket True Retrieve source False\n",
+      "rev 3\n",
+      "______________________\n",
+      "Execution 11 self other bucket Cross bucket True Retrieve source False\n",
+      "rev 3\n",
+      "______________________\n",
+      "Execution 12 id same bucket Cross bucket False Retrieve source False\n",
+      "rev 3\n",
+      "______________________\n",
+      "Execution 13 self same bucket Cross bucket False Retrieve source False\n",
+      "rev 3\n",
+      "______________________\n",
+      "Execution 14 id other bucket Cross bucket False Retrieve source False\n",
+      "<action> catch_http_error\n",
+      "<error> RetrievalError: 404 Client Error: Not Found for url: https://bbp.epfl.ch/nexus/v1/resources/dke/kgforge/_/http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FGO_0038048\n",
+      "\n",
+      "Not found\n",
+      "______________________\n",
+      "Execution 15 self other bucket Cross bucket False Retrieve source False\n",
+      "<action> retrieve\n",
+      "<error> RetrievalError: Provided resource identifier https://bbp.epfl.ch/nexus/v1/resources/neurosciencegraph/datamodels/_/http:%2F%2Fpurl.obolibrary.org%2Fobo%2FGO_0038048 is not inside the current bucket, use cross_bucket=True to be able to retrieve it\n",
+      "\n",
+      "Not found\n",
+      "______________________\n"
+     ]
+    }
+   ],
+   "source": [
```

Comment:
It would be nice to have a small explanation on what is printed, even when the code may be self-explanatory, it's always nice to have some words as intro